packages/atlas/lib/src/provider.dart

```diff
@@ -29,6 +29,7 @@ abstract class Provider {
     final ArgumentCallback<AtlasController> onMapCreated,
     final ArgumentCallback<CameraPosition> onCameraPositionChanged,
     final ArgumentCallback<LatLng> onLocationChanged,
+    final ArgumentCallback<bool> onPan,
```

Comment:
Thoughts on renaming to onCameraMove and changing to ArgumentCallback<CameraPosition>?